lib/mix/test/mix/tasks/test_test.exs

```diff
@@ -46,21 +52,21 @@ defmodule Mix.Tasks.TestTest do
   defp ex_unit_opts_from_given(passed) do
     passed
     |> ex_unit_opts()
-    |> Keyword.drop([:failures_manifest_file, :autorun])
+    |> Keyword.drop(@default_opts)
   end
 
   test "--stale: runs all tests for first run, then none on second" do
-    in_fixture "test_stale", fn ->
+    in_fixture("test_stale", fn ->
       assert_stale_run_output("2 tests, 0 failures")
 
       assert_stale_run_output("""
       No stale tests
       """)
-    end
+    end)
   end
 
   test "--stale: runs tests that depend on modified modules" do
-    in_fixture "test_stale", fn ->
+    in_fixture("test_stale", fn ->
```

Comment:
Whatever you are using to format the code is not considering the .formatter.exs at the root of the Elixir codebase, as it should not add those parens. :)

Reply:
Ah. Yeah it seems like it's coming from my Sublime Text formatter. I'll remove these changes